wwwapp/views.py

```diff
@@ -510,15 +514,17 @@ def unregister_from_workshop_view(request):
 
 
 @permission_required('wwwapp.export_workshop_registration')
-def data_for_plan_view(request):
+def data_for_plan_view(request, year: int) -> HttpResponse:
```

Comment:
Umm did you somehow accidentally pull my commits into this or was this on purpose for some reason?